src/wasm_runtime/src/component.rs

```diff
@@ -43,10 +44,12 @@ static CUR_INSTANCE: Mutex<Option<Instance>> = Mutex::new(None);
 hyperlight_wasm_macro::wasm_guest_bindgen!();
 
 // dummy for compatibility with the module loading approach
+#[instrument(skip_all, level = "Info")]
```

Comment:
should these all be info or a different level like debug?

Reply:
do you think an end user of hl-wasm would care about this information?  Or is this mostly for us?